src/node_modules/logger.js

```diff
@@ -71,7 +68,7 @@ const requirePeer = codependency.register(module);
 
 const nonConfigurable = (target, name, descriptor) => {
 	descriptor.configurable = false;
-	descriptor.enumerable = true;
+	descriptor.enumerable   = true;
```

Comment:
I really do not think is more readable, especially, when we have a considerable gap between variable and operator. 

But, this is my point of view...